blog/2024-03-07-cloud-storage/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+slug: cloud-storage
+title: |
+  Cloud storage: gain flexibility and performance for data access for your project in Renku
+authors: [elisabet]
+tags:
+  - Cloud Storage
+  - Data
+  - RenkuLab
+  - Renku CLI
+image: ./image_1.png
+---
+
+![Cloud storage access from Renku](./image_1.png)
+
+Working with your data in Renku is faster and simpler: you can write and read your data from your external storage services, such as S3, Azure Blob or WebDAV among others.
+
+RenkuLab interactive sessions can access data hosted in a cloud storage system, and the storage is simply mounted as another folder in your session. You can concentrate on developing your code to analyse your the data and share the results with your colleagues or even with the broader community.
```

Comment:
```suggestion
RenkuLab interactive sessions can access data hosted in a cloud storage system, and the storage is simply mounted as another folder in your session. You can concentrate on developing your code to analyse your data, rather than worrying about moving data around! Plus, when you add a cloud storage to your project, that configuration is available to all project members, so you can share the results with your colleagues or even with the broader community. Who has access is still completely in your control: the access to cloud storage is controlled by the storage provider, not Renku.
```